src/main/java/org/opentripplanner/routing/algorithm/filterchain/deletionflagger/RemoveItinerariesWithShortStreetLeg.java

```diff
@@ -0,0 +1,45 @@
+package org.opentripplanner.routing.algorithm.filterchain.deletionflagger;
+
+import java.util.List;
+import org.opentripplanner.model.plan.Itinerary;
+import org.opentripplanner.model.plan.Leg;
+import org.opentripplanner.routing.algorithm.filterchain.ItineraryListFilter;
+import org.opentripplanner.street.search.TraverseMode;
+
+/**
+ * This filter is useful if you want to remove those results where there is a short bicycle
+ * leg followed by parking the bike and taking transit. In such a case you would not need a bike
+ * could just walk to the stop instead.
+ */
```

Comment:
Should we transform the itinerary in those cases, or does the user need to do a new search for that?

Reply:
Well, it's specifically geared towards digitransit, which will show the walk,transit results in a different tab. It would be unexpected to have them show up in the bike tab.